x-pack/legacy/plugins/apm/e2e/cypress/integration/apm.feature

```diff
@@ -0,0 +1,9 @@
+Feature: APM
+
+  Background:
+    Given a user browses the APM UI application
+    When the user inspects the opbeans-go service
```

Comment:
@MadameSheema how do you see having a `When` clause in a `Background`? I'm not completely sure if moving this line to the Scenario itself would be a more accurate approach. wdyt?

Reply:
Hi!! Sorry for my late reply, I was a little bit busy. I don't like to use Backgrounds at all. This is because if you have a big file in a feature you can skip it when reading it.

Reply:
A part from what I said in the previous comment. If you only have one scenario, it does not make sense to have a background at all :)

Reply:
I was preparing for future updates... but I agree with you :) I'll send a commit moving the background step to the scenario. Thanks!!!